spec/models/user_access_spec.rb

```diff
@@ -70,13 +70,14 @@
 
         it "returns the organizations an admin can perform actions on" do
           permission_matrix.each do |admin, action, current_resource, expected_resources|
-            admin.accesses.create(resource: current_resource)
+            admin.accesses.create!(resource: current_resource)
 
             expect(admin.accessible_organizations(action)).to match_array(expected_resources),
               error_message(admin,
                 action,
                 expected_resources,
                 admin.accessible_organizations(action))
+            admin.accesses.delete_all
```

Comment:
We have to manually clear out the user_accesses so the next iterations in the matrix succeed.